src/screens/HomeScreen.tsx

```diff
@@ -63,7 +63,7 @@ const HomeScreen = () => {
 		return () => {
 			linkingEvent.remove();
 		};
-	}, [onInitClient, onHandleDeepLink]);
+	}, []);
```

Comment:
I think the dependencies here were fine, they are memoized methods in the same component. So I think you don't need to remove them!